docs/teleport/aks-getting-started.md

```diff
@@ -1,358 +1,222 @@
-- k8s version 1.19.7 or later is required, as Project Teleport depends on containerd.
-- ACR and AKS must be in the same regions. This is less of a limitation, rather a design constraint. It's always a best practice to have the content required for deployment to be within the same region. Project Teleport depends on this standard to mount layers within an Azure network, regional boundary.
-- To manage storage costs, geo-replication is not yet supported. The ACR must be created in the same region as the AKS cluster. Future versions will support geo-replication.
-- VNet support is not yet enabled.
+## Limitations
 
-## Set environment variables
+* Requires node pools in your AKS cluster to use Kubernetes 1.19.0 or greater.
+* To use Project Teleport with ACR and AKS, your node pools must use `contanerd` as the container runtime. AKS clusters with node pools using Kubernetes before 1.19.0 use Moby as the container runtime.
+* Each ACR used must have Project Teleport enabled.
+* Each ACR must use the *Premium* Tier and be in the same region as your AKS cluster. See [Project Teleport supported regions][teleport-regions].
+* At this time, Project Teleport supports Linux containers on AKS clusters. Windows support is not yet available.
 
-Configure variables unique to your environment. Note the ACR and AKS instances must both be in one of the [teleport supported regions][teleport-regions].
+### Enable Project Teleport on an ACR
 
-```azurecli-interactive
-AKS=myaks
-AKS_RG=${AKS}-rg
-RG_LOCATION=westus2
-K8S_VERSION=1.19.7
-ACR=myacr
-ACR_URL=${ACR}.azurecr.io
-```
-
-## Create an AKS Cluster
-
-At the current moment, Teleport does not yet support managed identity access to teleport expanded layers. Until managed identities are supported, configure the cluster with a service principal.
-
-```azurecli-interactive
-SP_PWD=$(
-  az ad sp create-for-rbac \
-  --skip-assignment \
-  --name ${AKS}-sp 
-  --query password -o tsv)
-
-az aks create \
-    -g ${AKS_RG} \
-    -n ${AKS} \
-    --attach-acr $ACR \
-    --kubernetes-version ${K8S_VERSION} \
-    -l $RG_LOCATION \
-    --service-principal $(az ad sp show \
-        --id http://${AKS}-sp \
-        --query appId \
-        --output tsv) \
-    --client-secret $SP_PWD
-
-az aks get-credentials \
-    -g ${AKS_RG} \
-    -n ${AKS}
-```
+Create a [premium instance][acr-tiers] of Azure Container Registry in one of the [Project Teleport supported regions][teleport-regions].
 
-A standard AKS cluster should now exist. Verify with:
+Sign up for ACR Project Teleport using [the signup form][teleport-signup-form] and provide the resource ID of the ACR. Once enabled, you'll receive an email confirming your ACR has been enabled for Project Teleport.
 
-```azurecli-interactive
-kubectl get nodes
-```
+Once you receive a confirmation email, push an image to a new repository. You may also use `az acr import` to copy an image from another registry.
```

Comment:
It might be nice to note that existing repositories are not expanded automatically?